### PR TITLE
cmd/juju: Use the new visitwebpage function to allow usso login via the cli

### PR DESCRIPTION
**Pull request by mattyw**
(Review request: http://reviews.vapour.ws/r/4063/)

### CODE REVIEW

File: jujuclient/oauth.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+package jujuclient
+
+import (
+	"github.com/juju/juju/juju/osenv"
+)
+
+// StoreOAuthToken is the location where the charm store oauth token is
+// expected to be found.
+func StoreOAuthToken() string {
```
Comment (alesstimec) [inline jujuclient/oauth.go]: nitpick: StoreOAuthTokenLocation

**Comment by alesstimec** [comment]
LGTM! well done!

**Comment by mattyw** [comment]
$$merge$$

**Comment by mattyw** [comment]
$$merge$$

**Comment by bz2** [comment]
Build failed: Matty keeps asking for it to be cancelled.

**Comment by bz2** [comment]
$$merge$$

**Comment by mattyw** [comment]
$$merge$$

**Pull request merged**